src/main/java/org/opentripplanner/standalone/config/NodeAdapter.java

```diff
@@ -168,6 +168,11 @@ public String asText(String paramName, String defaultValue) {
         return param(paramName).asText(defaultValue);
     }
 
+    public List<String> asTexts(String paramName, List<String> defaultValue) {
+        if(!exist(paramName)) return defaultValue;
+        return arrayAsList(paramName, JsonNode::asText);
+    }
+
```

Comment:
This 2nd argument of this method should be a Set, and the return should be Set so it matches the need. 

Why?
  - Removes `.stream().collect(Collectors.toSet())` from the NetexConfig witch shuld be as easy to read as possible pushing the details down into the NodeAdaptor.
  - Simplify unit testing - only the NodeAdaptor have logic and needs to be unit tested.
  - In the future we could use this code to generate example default config and documentation, but if you do it the way it is done the documentation would miss the fact that you want a set. 
  - We can add generic validation for the return type, like log warning for duplicate entries in the config.

Reply:
But doesn't this class mirror the JSON types? In JSON there isn't a Set type so `['aa', 'aa']` would be converted to `['aa']`. Besides the `asDoubles` method in `NodeAdapter` returns a `List<Doubles>`.

How about I rename it to `asTextSet` and make it return a set?

Reply:
I went with the rename. I hope this is ok.

Reply:
`asTextSet ` is good.

The purpose of the `NodeAdapter` is to extend the behavior of what we get from the JSON library, this include adding additional restrictions and parsing data into type-safe classes. The changes you did looks good.